site/source/docs/getting_started/downloads.rst

```diff
@@ -108,6 +108,8 @@ Windows
 
 #. Install Python 2.7.12 or newer (older versions may not work due to `a GitHub change with SSL <https://github.com/emscripten-core/emscripten/issues/6275>`_).
 
+  .. note:: You can validate Python installation by ``Python -V``, if this commend not write anything you have set path to Python Windows Mini App (This it's Windows default behaviour. It does 2 thing: open Microsoft Store with "python" search, and eats all python errors, so commands like ``./emsdk install latest`` doesn't write any error but also doesn't do anything)
```

Comment:
Thanks!

I'm not sure I understand this, though - what is `Python Windows Mini App`?

Also I have a few text suggestions:

```suggestion
  .. note:: You can validate the Python installation by ``Python -V``. If this command does not write anything then you have to set the path to the Python Windows Mini App (this is Windows' default behaviour, it does 2 things: open Microsoft Store with "python" search, and silently eats all python errors, so commands like ``./emsdk install latest`` don't write any error but also don't do anything).
```